apps/playwright/tests/admin/pool-candidate.spec.ts

```diff
@@ -94,6 +96,208 @@ test.describe("Pool candidates", () => {
     candidate = application;
   });
 
+  test("Snapshot content is rendered", async ({ appPage }) => {
+    await loginBySub(appPage.page, "admin@test.com");
+    await appPage.page.goto(`/en/admin/candidates/${candidate.id}/application`);
+    await appPage.waitForGraphqlResponse("PoolCandidateSnapshot");
+
+    // check verification appears
+    await expect(
+      appPage.page
+        .locator("div")
+        .filter({ hasText: /^Priority statusThis claim is unverified\.$/ })
+        .getByRole("paragraph"),
+    ).toBeVisible();
+    await expect(
+      appPage.page
+        .locator("div")
+        .filter({ hasText: /^Veteran statusThis claim is unverified\.$/ })
+        .getByRole("paragraph"),
+    ).toBeVisible();
+
+    // expand snapshot
+    await appPage.page
+      .getByRole("button", { name: "Expand all application" })
+      .click();
+
+    // personal information
+    await expect(appPage.page.getByText(/Given namePlaywright/i)).toBeVisible();
+    await expect(
+      appPage.page
+        .getByLabel("Personal and contact")
+        .getByText(/playwright.sub./),
+    ).toBeVisible();
```

Comment:
Can we also ensure this has the proper role? You can do something like:

```tsx
const personal = appPage.page.getByRole("region", { name: /personal and contact/i });
await expect(personal).toContainText(/given nameplaywright/i);
await expect(personal).toContainText(/playwright.sub./);
```

This will also work with the other accordions as well.

Reply:
Oh cool, I didn't know about the `region` thing
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12182/commits/42cbc8fb1b83f07840605bcda6286c7adeac8815